pkg/secrethub/repo.go

```diff
@@ -309,3 +316,113 @@ func (c *Client) getRepoIndexKey(repoPath api.RepoPath) (*crypto.SymmetricKey, e
 
 	return repoIndexKey, nil
 }
+
+// Iterator returns a new iterator that retrieves all repos in the given namespace.
+func (s repoService) Iterator(namespace string, params *RepoIteratorParams) RepoIterator {
+	return &repoIterator{
+		iterator: iterator.New(
+			iterator.PaginatorFactory(
+				func() ([]interface{}, error) {
+					repos, err := s.List(namespace)
+					if err != nil {
+						return nil, err
+					}
+
+					res := make([]interface{}, len(repos))
+					for i, element := range repos {
+						res[i] = element
+					}
+					return res, nil
+				},
+			),
+		),
+	}
+}
+
+// IteratorMine returns an iterator that retrieves all repos of the current user.
+func (s repoService) IteratorMine(_ *RepoIteratorParams) RepoIterator {
+	return &repoIterator{
+		iterator: iterator.New(
+			iterator.PaginatorFactory(
+				func() ([]interface{}, error) {
+					repos, err := s.ListMine()
+					if err != nil {
+						return nil, err
+					}
+
+					res := make([]interface{}, len(repos))
+					for i, element := range repos {
+						res[i] = element
+					}
+					return res, nil
+				},
+			),
+		),
+	}
+}
+
+// AccountIterator returns a new iterator that retrieves all accounts in the given namespace.
+func (s repoService) AccountIterator(namespace string, params *RepoIteratorParams) AccountIterator {
```

Comment:
`ListAccounts` has a path as argument, while this iterator has a namespace as argument. How does that work?

Reply:
That is another typo.